config/locales/devise.en.yml

```diff
@@ -33,6 +33,7 @@ en:
         title: "Forgot your password?"
       edit:
         title: "Change your password"
+      invalid_token: "The password reset session you've loaded is outdated. Please start the password reset process again."
```

Comment:
Too wordy.

`The link used for your request has expired. Please restart the password change process.`